packages/runtime/src/controls/icon.ts

```diff
@@ -0,0 +1,8 @@
+export const unstable_Icon = {
```

Comment:
I was thinking that we would eventually want an enum for all our icon components. Open to feedback

Reply:
Agreed (changed)